features/step_definitions/support_action_steps.rb

```diff
@@ -4,6 +4,15 @@
   choose_yes_on_support_action(page, support_action)
 end
 
+When('I add support actions') do
+  visit "/contacts/#{@contact.id}"
+  click_link 'Add Needs'
```

Comment:
Oh shoot, did we leave the word 'Needs' on this list? We should probably change it to 'Add support actions' shouldn't we 😬